ai-data/llm-inference/how-to/delete-deployment.mdx

```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: How to delete a LLM Inference deployment
+  description: This page explains how to deletee a LLM Inference deployment
+content:
+  h1: How to delete a LLM Inference deployment
+  paragraph: This page explains how to deletee a LLM Inference deployment
+tags: ai deleteing
+dates:
+  validation: 2024-03-06
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Once you have finished your inference tasks you can delete your deployment from the Scaleway console.
+
+<Macro id="iam-requirements" />
```

Comment:
This will be replaced with <Macro id="requirements" />

Reply:
Macro id="requirements"

Reply:
```suggestion
<Macro id="requirements" />
```